java/pypowsybl/src/main/java/com/powsybl/python/rao/RaoDataframes.java

```diff
@@ -67,7 +83,13 @@ public record AngleCnecResult(String cnecId, Instant instant, String contingency
 
     public record VoltageCnecResult(String cnecId, Instant instant, String contingency, TwoSides side, double minVoltage, double maxVoltage, double margin) { }
 
-    public record ActivatedRemedialActionResult(String remedialActionId, Instant instant, String contingency, double optimizedTap, double optimizedSetPoint) { }
+    public record ActivatedRemedialActionResult(String remedialActionId, Instant instant, String contingency) { }
```

Comment:
Maybe use a single "ActivatedActionResult" record, instead of two identical records